CHANGELOG.md

```diff
@@ -25,7 +25,8 @@ Next
 - Suppress duplicate reports in count mode with -q, --quiet or -Q, --squiet
   (#392, thanks @gsnw-sebast and @auerswal)
 - Switch to alpine-based multi-stage Docker build to reduce image size and improve build performance
-  Add OpenContainers-compatible labels (#399)
+  Add OpenContainers-compatible labels
+  Add fping man page (#399)
```

Comment:
Since the current Debian-based container uses `make install`, the man page is also included there. Thus I'd prefer if you did not add the man page to the change log.